src/ghdcbot/core/social_validators.py

```
@@ -0,0 +1,253 @@
+        # Remove @mention prefix if present
+        if value.startswith("@"):
+            value = value[1:]
+        
+        # Extract username from URL
+        if "x.com/" in value or "twitter.com/" in value:
+            # Parse URL
+            parts = value.split("/")
+            # Find the part after x.com or twitter.com
+            for i, part in enumerate(parts):
+                if part in ("x.com", "twitter.com"):
+                    if i + 1 < len(parts):
```

Ruff (0.15.18)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)